text/0000-lock-without-registry.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-09
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The lockfile yarn.lock should not include the base registry (`https://registry.npmjs.org`).
+
+# Motivation
+
+In yarn.lock, the `resolved` field includes registry such as `https://registry.npmjs.org`.
+In China, most developers will set it to `https://registry.npm.taobao.org` for speed; but it seems slow for travis-ci and circleci.
+
+# Detailed design
+
+Replace the `resolved` by a `hash` field.
```

Comment:
In a way,  the whole point of this RFC to discuss do that: constructing the URL on the fly :)

When you say it is the npm backend constructing the URL, do you mean that IF we wanted to reconstruct it, we would need to store in the lockfile that it is a npm dependency so that we know we need the npm backend to reconstruct the full URL?